radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/data/ApiErrors.java

```diff
@@ -102,6 +103,6 @@ public String message() {
 
 	@Override
 	public int code() {
-		return code;
+		return -2000 - ordinal();
```

Comment:
I'd extract -`2000` into a variable with a name explaining what it means

Reply:
This is a preliminary step for reorganizing all errors - my next PR.